internal/pkg/describe/backend_service.go

```diff
@@ -20,7 +21,7 @@ type BackendServiceDescriber struct {
 	service         *config.Service
 	enableResources bool
 
-	store                storeSvc
+	store                deployStoreSvc
```

Comment:
i'll rename this back to be "storeSvc" once i fix env show in my next pr. Then we'll remove dependency to config pkg.